app/Mage.php

```diff
@@ -526,7 +526,7 @@ public static function getModel($modelClass = '', $arguments = [])
      *
      * @param   string $modelClass
      * @param   array $arguments
-     * @return  Mage_Core_Model_Abstract
+     * @return  Mage_Core_Model_Abstract|false
```

Comment:
@tmotyl adding the "|false" triggered quite some other things, should we add them to the baseline or try to fix them?